tools/checked-c-convert/ProgramInfo.cpp

```diff
@@ -63,7 +66,7 @@ PointerVariableConstraint::PointerVariableConstraint(const Type *_Ty,
   BaseType = tyToStr(Ty);
 
   // Special case for void to not make _Ptr<void> pointers.
-  if( BaseType == "void" ) 
+  if( BaseType == "void" || BaseType == "struct __va_list_tag *" ) 
```

Comment:
I agree. I looked for this, but couldn't find it. varargs is not a "builtin" type, it's introduced as something else (the list of builtins is enumerated [here](https://github.com/Microsoft/checkedc-clang/blob/master/include/clang/AST/BuiltinTypes.def). I'll make a work item (#61) for it. I can deal with the void case, though.